src/modules/options/pivotTableConfig.js

```diff
@@ -82,6 +83,7 @@ export default ({
             key: 'style-section-1',
             content: React.Children.toArray([
                 <Title label={i18n.t('Table title')} />,
+                <Subtitle dataTest="pivot-table-subtitle" />,
```

Comment:
Perhaps not, but without it we ended up with `data-test="undefined-input`. While checking this out in the browser, I also realised that the text field was missing a label. So I made some changes:
- The subtitle now accepts a `label` prop
- The `dataTest` prop now has a default value so we don't need to set anything from the consuming component to prevent `data-test="undefined-input`